packages/generate/dbschema/default.esdl

```diff
@@ -1,209 +1,3 @@
-using extension pgvector;
-
 module default {
 
-  scalar type Genre extending enum<"Horror", "Action", "RomCom", "Science Fiction", "Select">;
```

Comment:
Most of this file moved to `integration-lts` except for the pgvector related stuff which ended up in `integration-stable`.